core/server/common/src/main/java/alluxio/master/journal/checkpoint/CheckpointType.java

```diff
@@ -26,7 +26,24 @@
    * This format uses kryo's chunked encoding to write multiple [checkpoint_name, checkpoint_bytes]
    * entries to the output stream.
    */
-  COMPOUND(1);
+  COMPOUND(1),
+  /**
+   * A series of longs written by {@link java.io.DataOutputStream}.
+   */
+  LONGS(2),
+  /**
+   * A RocksDB backup in .tar.gz format.
+   */
+  ROCKS(3),
+  /**
+   * This format sequentially writes delimited InodeMeta.Inode protocol buffers one after another
+   * using the protocol buffer writeDelimitedTo method.
+   */
+  INODE_PROTOS(4),
+  /**
+   * A checkpoint consisting of a single long value written by a data output stream.
+   */
+  LONG(5);
```

Comment:
Could this be a special case of `LONGS`?

Reply:
It could, it would just be less typesafe. I'd prefer to consider `LONG` and `LONGS` as two separate types